pendulum/date.py

```diff
@@ -207,10 +207,15 @@ def is_birthday(self, dt=None):
         if dt is None:
             dt = Date.today()
 
-        instance = dt1 = self.__class__(dt.year, dt.month, dt.day)
+        instance = self.__class__(dt.year, dt.month, dt.day)
 
         return (self.month, self.day) == (instance.month, instance.day)
 
+    # the additional method for checking if today is the anniversary day
+    # the alias is provided to start using a new name and keep the backward compatibility
+    # the old name can be completely replaced with the new in one of the future versions
+    is_anniversary = is_birthday
```

Comment:
I would make the alias the other way round, i.e. `is_anniversary` is the original method with `is_birthday` as the copy

Reply:
Good point, I will change it as you proposed 👍